# This ebuild was auto-generated by homeassistant script, and the script was not able to find a better candidate.
EAPI=8

DISTUTILS_USE_PEP517=setuptools
PYTHON_COMPAT=( python3_{12,13{,t}} )
PYPI_NO_NORMALIZE=1
GENERATED_IUSE="all external-authentication-token-cache gssapi kerberos sqlalchemy tests"
IUSE="${GENERATED_IUSE}"

inherit distutils-r1 pypi

DESCRIPTION=""
HOMEPAGE="
  https://pypi.org/project/trino/"

LICENSE=""
SLOT="0"
KEYWORDS="amd64 arm64"

GENERATED_DEPEND="
	tests? ( dev-python/black[${PYTHON_USEDEP}] )
	tests? ( dev-python/boto3[${PYTHON_USEDEP}] )
	tests? ( <dev-python/httpretty-1.1[${PYTHON_USEDEP}] )
	tests? ( dev-python/isort[${PYTHON_USEDEP}] )
	external-authentication-token-cache? ( dev-python/keyring[${PYTHON_USEDEP}] )
	tests? ( dev-python/keyring[${PYTHON_USEDEP}] )
	gssapi? ( ~dev-python/krb5-0.5.1[${PYTHON_USEDEP}] )
	tests? ( ~dev-python/krb5-0.5.1[${PYTHON_USEDEP}] )
	dev-python/lz4[${PYTHON_USEDEP}]
	tests? ( dev-python/pytest[${PYTHON_USEDEP}] )
	tests? ( dev-python/pytest-runner[${PYTHON_USEDEP}] )
	dev-python/python-dateutil[${PYTHON_USEDEP}]
	dev-python/pytz[${PYTHON_USEDEP}]
	>=dev-python/requests-2.31.0[${PYTHON_USEDEP}]
	gssapi? ( dev-python/requests-gssapi[${PYTHON_USEDEP}] )
	tests? ( dev-python/requests-gssapi[${PYTHON_USEDEP}] )
	all? ( dev-python/requests-kerberos[${PYTHON_USEDEP}] )
	kerberos? ( dev-python/requests-kerberos[${PYTHON_USEDEP}] )
	tests? ( dev-python/requests-kerberos[${PYTHON_USEDEP}] )
	all? ( >=dev-python/sqlalchemy-1.3[${PYTHON_USEDEP}] )
	sqlalchemy? ( >=dev-python/sqlalchemy-1.3[${PYTHON_USEDEP}] )
	tests? ( >=dev-python/sqlalchemy-1.3[${PYTHON_USEDEP}] )
	tests? ( dev-python/testcontainers[${PYTHON_USEDEP}] )
	dev-python/tzlocal[${PYTHON_USEDEP}]
	dev-python/zstandard[${PYTHON_USEDEP}]
	tests? ( dev-vcs/pre-commit[${PYTHON_USEDEP}] )
"
RDEPEND="${GENERATED_DEPEND}"

distutils_enable_tests pytest